test/visualTest.ts

```diff
@@ -375,7 +375,8 @@ describe("AsterPlot", () => {
             beforeEach(() => {
                 dataView.metadata.objects = {
                     outerLine: {
-                        show: true
+                        show: true,
+                        showStraightLines: true
```

Comment:
default value for this setting is already `true`, do you want to assign it here?